### DISCUSSION TITLE
If anybody wants to opt the enterprise edition then will it be only for Pim 11 or this can be opted on Pim 10.4 version as well

### DISCUSSION THREAD

**Discussion by rajeshkumarhm** [pimcore/pimcore#16087] [Q&A / Support]
If anybody wants to opt the enterprise edition then will it be only for Pim 11 or this can be opted on Pim 10.4 version as well?

**Comment by kingjia90** [comment]
By looking at the LTS https://pimcore.com/en/services/lts (included in the enterprise subscription), it should be covering both 11 and 10.
Please contact the sales team in the form below that link for more precise info.

**Reply by dvesh3** [reply]
you can also check for supported versions on LTS [here](https://github.com/pimcore/pimcore#supported-versions)
